src/components/navigation/problem-tab-content.tsx

```diff
@@ -59,13 +71,15 @@ export const ProblemTabContent: React.FC<IProps>
         {isTeacher && showSolutionsSwitch &&
           <SolutionsButton onClick={handleToggleSolutions} isToggled={showTeacherContent} />}
       </div>
-      {sections.map((section) => {
-        return (
-          <TabPanel key={`section-${section.type}`} data-focus-section={section.type}>
-            <ProblemPanelComponent section={section} key={`section-${section.type}`} />
-          </TabPanel>
-        );
-      })}
+      <div className="problems-panel" style={problemsPanelStyle}>
```

Comment:
```suggestion
      <div className="problem-panel" style={problemPanelStyle}>
```
It's showing multiple sections for a single problem, I believe.